src/main/py/peer/peer.py

```diff
@@ -222,21 +226,58 @@ async def send_trade_response(
     async def handle_accept_trade(self, response: OfferResponse):
         """Handle an accepted trade"""
 
-        self.logger.info(response)
+        trade_confirmation = TradeConfirmation(
+            response.trade_id,
+            self.inventory.pending_trades[response.trade_id].get_artwork_key(),
+            self.keys["public"],
+            True,
+        )
+        try:
+            set_success = await self.node.set(
+                utils.generate_random_sha1_hash(), pickle.dumps(trade_confirmation)
+            )
+            if set_success:
+                self.logger.info("Fragment sent")
+                self.inventory.remove_owned_artwork(self.inventory.pending_trades[response.trade_id].get_artwork_key())
```

Comment:
Please run `make` before committing so formatting is taken care of by `ruff` in the `test` target. This line is a bit long.

It might be better solved though by pulling the parameter into a local var, especially in this case since the param is also used to create the `TradeConfirmation`.